drivers/SmartThings/matter-lock/profiles/nonfunctional-lock.yml

```diff
@@ -0,0 +1,14 @@
+name: nonfunctional-lock
```

Comment:
this could just be "lock without codes" like we have for z-wave and zigbee, yeah?

Reply:
Well its actually "lock without lock". It is missing the Lock capability since the lock commands wont work. It has lock codes because as you mention elsewhere we want to allow the user to delete/manage codes.

Reply:
reading is hard